packages/core/src/components/Form.js

```diff
@@ -382,6 +387,19 @@ export default class Form extends Component {
     );
   };
 
+  getFirstErrorField = error => {
+    const firstErrorField = error?.property?.replace(".", "");
```

Comment:
this only works for single level object fields

```
error?.property?.replace(".", "")
          .replace(/\./g, "_")
          .replace(/\[/g, "_")
          .replace(/\]/g, "")
```

I think covers nested fields and array items.  There might be a better way to get the id from the schema fields - I haven't had a chance t look deep enough for that yet